kore/src/Kore/Repl/Interpreter.hs

```diff
@@ -200,6 +202,37 @@ showLeafs = do
     showPair :: (NodeStates, [Graph.Node]) -> String
     showPair (ns, xs) = show ns <> ": " <> show xs
 
+showPrecBranch
+    :: MetaOrObject level
```

Comment:
I think that, unless we need to do something with the configuration (like `showConfig` displays it), we shouldn't need the `MetaOrObject` constraint. Can you check whether it compiles without?